internal/scorecard/plugins/plugin_runner.go

```diff
@@ -346,3 +348,47 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 	}
 	return output, nil
 }
+
+func ListInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig) (scapiv1alpha1.ScorecardOutput, error) {
+	var suites []schelpers.TestSuite
+
+	if pluginType == BasicOperator {
```

Comment:
WDYT about use switch here and add the logic to create/build the object in the NewBasicTestSuite(params) and NewOLMTestSuite(params) and? Who knows more about the object then itself ;-)?

Reply:
ok